### PR TITLE
Doc 6628 style guide work

### PR DESCRIPTION
**Pull request by paperclypse**
### Tell us why

As part of importing the style guide content, a lot of things got moved around and tweaked. There were some broken links and some procedures that needed to be updated.

### CODE REVIEW

**Comment by bradleycamacho** [comment]
I can take a look at this in ~20 minutes unless someone beats me to it!

**Comment by paperclypse** [comment]
Thanks @bradleycamacho!

File: src/content/docs/style-guide/processes-procedures/delete-document.mdx [hunk 1]
```diff
@@ -10,12 +9,12 @@ redirects:
...
+  When you delete a document, its content and its redirects will still be available in the GitHub commit history. In general, if you're not on the New Relic Docs team, please don't delete any docs. Instead, [file an issue](https://github.com/newrelic/docs-website/issues/new/choose) and someone on our team will help you out.
 </Callout>
 
 If you are certain you want to delete a document, do this:
 
-1. Locate the doc you want to delete. 
+1. Locate the MDX file of the doc you want to delete. 
 2. Save the doc's content in a file and attach it to an associated Jira (in case we need the doc again).
-3. PENDING: INFO ON REDIRECTS
+3. Find the most relevant doc and [add the deleted doc's URL as a redirect](/docs/style-guide/rename-or-redirect-document).
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/processes-procedures/delete-document.mdx:19]: Is this missing a step to delete it from the `.yml` files?

**Review by bradleycamacho — APPROVED** [review]
Looks great! Left a single comment and I'm pushing a commit onto this to add the Tech Writer Workflow doc to the taxonomy.

File: src/content/docs/style-guide/processes-procedures/delete-document.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/style-guide/processes-procedures/delete-document.mdx:19]: Yup! Great catch, @bradleycamacho! I'll get that added.

**Pull request merged**